tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -51,6 +51,19 @@ def if_then_validator_func(value):
 
     return lambda value: if_then_validator_func(value)
 
+def sumIsEqual(condition_field, sum_fields=[]):
+    """Validate that the sum of the sum_fields equals the condition_field."""
+    def sumIsEqualFunc(value):
+        sum = 0
+        for field in sum_fields:
+            sum += value[field] if type(value) is dict else getattr(value, field)
+
+        condition_val = value[condition_field] if type(value) is dict else getattr(value, condition_field)
+        return (True, None) if sum == condition_val else (False,
+                                                          f"The sum of {sum_fields} does not equal {condition_val}.")
```

Comment:
@elipe17 is it possible to change `condition_val` to the data element label? the value is not as meaningful. 

![s3_val](https://github.com/raft-tech/TANF-app/assets/63075587/1b0c2c92-513d-432f-a212-052e1986cad9)